src/helpers/SkeletonHelper.js

```diff
@@ -77,6 +77,13 @@ class SkeletonHelper extends LineSegments {
 		 */
 		this.root = object;
 
+		/**
+		 * Reference to the skeleton object.
+		 *
+		 * @type {?Skeleton}
+		 */
+		this.skeleton = null;
```

Comment:
What line in `SkeletionUtils` is complaining about this missing property?

Reply:
I woud say this code is wrong. We have used to do this in the past but this was changed in #25811 and #25763.

Would you please revert the changes to `SkeletonHelper`?